src/Image/Darkroom/ImageMagick.php

```diff
@@ -72,8 +72,47 @@ protected function convert(string $file, array $options): string
 			}
 		}
 
+		// frame option to allow selecting layers for multi-layer or frames for animated images
+		$fileOptions = '';
+		$frame = $options['frame'];
+		$maxFrames = $this->frameCount($file, $options);
```

Comment:
I think this should happen inside the if clause so we don't run the extra command if no `frame` option has been set

Reply:
Ok I realised we need it for the `elseif` too. But then the name is misleading. This isn't max frames, but just the frame count. So maybe `frames` next to `frame` would be the better name